suites/pacific/cephadm/tier-1_multiple_subnet.yaml

```diff
@@ -0,0 +1,229 @@
+      polarion-id: CEPH-83574726
+      config:
+        steps:
+          - config:
+              service: host
+              command: set_address
+              args:
+                node: node1
+          - config:
+              service: host
+              command: label_add
+              args:
+                node: node1
+                labels: apply-all-labels
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: host
+                  address: true
+                  labels: apply-all-labels
+                  nodes:
+                    - node2
+                    - node3
+                - service_type: host
+                  address: true
+                  labels: apply-all-labels
+                  nodes:
+                    - node4
+                - service_type: host
+                  address: true
+                  labels: apply-all-labels
+                  nodes:
+                    - node5
+      abort-on-fail: true
+  - test:
+      name: Apply Monitor service
+      desc: Apply monitor service using label mon
+      module: test_mon.py
+      polarion-id: CEPH-83573772
+      config:
+        command: apply
+        service: mon
+        base_cmd_args:
+          verbose: true
+        args:
+          placement:
+            label: mon
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Apply Manager service
+      desc: Apply manager service using label mgr
+      module: test_mgr.py
+      polarion-id: CEPH-83573777
+      config:
+        command: apply
+        service: mgr
+        base_cmd_args:
+          verbose: true
+        args:
+          placement:
+            label: mgr
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Apply OSD Service
+      desc: Apply OSD service with all available devices
+      module: test_osd.py
+      polarion-id: CEPH-83573735
+      config:
+        command: apply
+        service: osd
+        base_cmd_args:
+          verbose: true
+        args:
+          all-available-devices: true
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: MDS Service deployment with spec
+      desc: Add MDS services using spec file
+      module: test_cephadm.py
+      polarion-id: CEPH-83574728
+      config:
+        steps:
+          - config:
+              command: shell
+              args:               # arguments to ceph orch
+                - ceph
+                - fs
+                - volume
+                - create
+                - cephfs
+          - config:
+              command: apply_spec
+              service: orch
+              validate-spec-services: true
+              specs:
+                - service_type: mds
+                  service_id: cephfs
+                  placement:
+                    label: mds
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
```

Comment:
do we need object?

Reply:
same answer to cover other service in multiple public network configured cluster,
Hence added RGW as well